### ISSUE TITLE
[1.3.2] Usage with Nuxt / document is not defined

### ISSUE THREAD

**Issue by Zsavajji** [RobinCK/vue-popper#31]
Hi, 
i've been using vue-popper with nuxt for a while with success, but this morning i upgraded the dependencies of my project, and updated to 1.3.2 from 1.3.0, and suddenly this error appears when rendering from the server.

```
ReferenceError: document is not defined at __vue_create_injector__ (/node_modules/vue-popperjs/dist/js/vue-popper.js:354:16)
```

Is this a bug or there's some new configuration for SSR which i'm missing out?

Thanks in advance

**Comment by RobinCK** [comment]
Hi @Zsavajji 
I think problem in rollup-plugin-vue
I downgrade plugin 
Please check 1.3.3 version

**Comment by Zsavajji** [comment]
Yes, sorry, never got back to you, 1.3.3 is working as expected!

**Comment by ayvanov** [comment]
Hi @RobinCK
Same error appears since version 1.4.1

**Comment by shlima** [comment]
+1

**Comment by shlima** [comment]
I can confirm, that using raw version from `/src/component/popler.js.vue` works perfect with SSR. @RobinCK could you please fix the issue ?

**Comment by shlima** [comment]
@RobinCK if you add the following line to webpack config:

```
globalObject: 'typeof self !== \'undefined\' ? self : this'
```

You package will be working in SSR mode, please see https://github.com/webpack/webpack/issues/6784#issuecomment-375941431

Could you please fix the issue?
